### ISSUE TITLE
how can i change the volume of selected video

### ISSUE THREAD

**Issue by ahsandeveloper1601** [wlanjie/trinity#68]
i want to change the volume of selected video which i have picked from the gallery

**Comment by wlanjie** [comment]
No, this function is not implemented, you need to add it yourself

**Comment by badcodercpp** [comment]
@zeropoint7877 you can do with a small hack , compile. ffmpeg along with this project, seperate audio stream and send audio stream to ffmpeg and set desired volume , like ``` ffmpeg -i input.wav -filter:a "volume=10dB" output.wav ```
